package.json

```
@@ -4,7 +4,7 @@
   "description": "A template for building Slack app Assistants",
   "main": "app.js",
   "scripts": {
-    "check": "npx tsc --checkJs --noEmit --esModuleInterop app.js",
+    "check": "npx tsc --checkJs --noEmit --esModuleInterop --skipLibCheck app.js",
```

```suggestion
    "check": "npx tsc --checkJs --noEmit --esModuleInterop --target esnext --module nodenext --moduleResolution nodenext app.js",
```

🦉 suggestion(blocking): We'll want to keep checking imported packages as a quick catch of changes happening with `@slack/bolt` releases too. Changes of https://github.com/slack-samples/bolt-js-assistant-template/pull/46 have more reference-